hw/usb/tinyusb/da146xx/include/tusb_hw.h

```diff
@@ -22,9 +22,28 @@
 
 #define CFG_TUSB_MCU OPT_MCU_DA1469X
 
+#include <os/mynewt.h>
 #include <syscfg/syscfg.h>
 #include <stdbool.h>
 
+#if MYNEWT_VAL(USB_MONITOR)
+/**
+ * Set event queue for USB monitoring
+ */
+void tinyusb_usb_monitor_evq_set(struct os_eventq *evq);
```

Comment:
this is not used anywhere

Reply:
It is only needed if the eventq for the callout should not be the default eventq.